app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -3,11 +3,11 @@
 
   <% if follow_request %>
     <% if follow_request.pending? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
+      <%= link_to follow_request, id: dom_id(follow_request), class: "btn btn-outline-secondary", remote: true, method: :delete do %>
```

Comment:
Hi Jan, your follow and unfollow is only working for the first post and this might be why. You need to make sure you are capturing the current sender and recipient id in order to pass it into your create js.erb and destroy.js.erb. 

Consider creating a div tag at the very to that captures follow_request sender.id and recipient.id of the post! Then passing that into your js files!